src/deserializers.ts

```diff
@@ -115,6 +124,269 @@ const deserializeInfoLayer = async (line: string): Promise<InfoLayerEntry | unde
 	// The data seems to be equal to info channel in 2.3.2
 	return deserializeInfoChannel(line)
 }
+const deserializeInfoConfig = async (line: string): Promise<InfoConfig> => {
```

Comment:
Should we perhaps split this whole file and move the contents into separate files in a folder?